shared/src/main/kotlin/security/auth/spi/WebSecurityMode.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2017/2021 e-voyageurs technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.tock.shared.security.auth.spi
+
+enum class WebSecurityMode {
```

Comment:
Actually, given that the input in TOCK Studio is a string (I don't think we can replace it with a `select` at the moment), and that anyone can provide their own implementation with a custom binding, does it still make sense to constrain the possible values to just these four?

I could imagine multiple JWT providers being present, and multiple web connectors each using their own.

Reply:
Yes, I agree with you, but today only one use case has been identified for this authentication, so I suggest you leave it as it is, and consider opening it up further in the future if other implementations come to be added.